src/utils/index.ts

```diff
@@ -332,6 +332,7 @@ export async function openURL(url?: string | null) {
 }
 
 export function isDashboardPreview(userAgent: object | undefined) {
+  return true;
```

Comment:
```suggestion
```

Reply:
Always make sure to check for any unnecessary code before submitting your review.

Reply:
oh my mistake. Thanks for the catch! Ill be careful of this.